src/picongpu/include/particles/particleToGrid/derivedAttributes/Counter.hpp

```diff
@@ -52,6 +52,6 @@ namespace derivedAttributes
         /* return attribute */
         return particleCounter;
     }
-} /* namespace derivedAttributes */
-} /* namespace particleToGrid */
-} /* namespace picongpu */
+}
```

Comment:
doesn't matter, I just add it and follow the herd until you all realize Benjamin was right :P